packages/runtime/test.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { tap, spec } = require('node:test/reporters')
```

Comment:
It will not work for nodejs 18, I guess
https://nodejs.org/api/test.html#test-reporters

Reply:
I tested it on Node v18